networks.json

```diff
@@ -9,8 +9,8 @@
     "4": {
       "events": {},
       "links": {},
-      "address": "0x9878669e883DDfC52620a6493b361213444cB974",
-      "transactionHash": "0x3745c5d7cb85b79f90140423699830969fb5a3514e7c41ef9be728f6ccdde6c2"
+      "address": "0xc2425e2B18cc3dF29BB9917bD3Cc7bCb4BCBde1A",
```

Comment:
I think it's usually nicer to have a new deploy or version number tag in a separate change. If we'd want to roll back this change it would now also revert an existing deployment and worse version number.